src/components/EventMap.js

```diff
@@ -40,7 +41,6 @@ export default function(store){
     watch: {
       events(newEvents, oldEvents) {
         this.plotEvents();
-        this.resetMap();
```

Comment:
This reset map stuff became unnecessary as a side effect of showing the event list on the right earlier in time.